core/audit/journal/log.go

```diff
@@ -13,11 +13,39 @@ package journal
 import (
 	"net/http"
 
-	"github.com/vmware-tanzu/secrets-manager/core/audit"
-	event "github.com/vmware-tanzu/secrets-manager/core/audit/state"
-	reqres "github.com/vmware-tanzu/secrets-manager/core/entity/reqres/safe/v1"
+	"github.com/vmware-tanzu/secrets-manager/core/audit/state"
+	"github.com/vmware-tanzu/secrets-manager/core/log/std"
 )
 
+type Entry struct {
+	CorrelationId string
+	Payload       string
```

Comment:
Simplified the audit log and added `Payload` as a catch-all attribute to dump any extra info that we might need.